rfc-139-enable-http3.md

```diff
@@ -0,0 +1,135 @@
+2. Each TCP connection is independent and have no knowledge of each other. All 6 are fighting for limited bandwidth which in turn can saturate the network.
+3. Many websites resorted to domain 'sharding', so as to allow a browser to open up more that 6 TCP connections, in turn exacerbating the problem mentioned in point 2.
+
+Another solution needed to be found.
+
+### HTTP/2
+HTTP/2 is a major iteration on the HTTP protocol. It started life as the [SPDY protocol](https://en.wikipedia.org/wiki/SPDY) created by Google, which was eventually used as the foundation for HTTP/2. One major shift in this version is the ability to multiplex stream assets over a single TCP connection. 
+
+Each TCP packet now includes a set of "frames" that contains additional meta information about each stream. Each asset is downloaded over a separate stream and then reconstructed by the browser. This allows multiple assets to be downloaded at the same time over a single TCP connection, removing the HOL blocking at the application (HTTP) layer. An example of this setup can be seen in the diagram below: 
+
+![An example of HTTP/2 frames within TCP packets.](rfc-139/5_H2_2files_multiplexed.png)
+
+Source: ["Head-of-Line Blocking in QUIC and HTTP/3: The Details"](https://calendar.perfplanet.com/2020/head-of-line-blocking-in-quic-and-http-3-the-details/) by [Robin Marx](https://twitter.com/programmingart).
+
+Unfortunately this didn't solve all the HOL blocking issues. Since HTTP/2 is built upon TCP, and TCP is a protocol that offers the reliable transmission of packets. TCPs job is to make sure that the packets leaving one device arrive at another device in the exact same order and without any missing packets. When packet loss occurs, this is when we again see HOL blocking. When a packet of data is lost over the network, this then holds up the whole (single) TCP connection. Packets behind the missing packet can't be processed until retransmission occurs.
+
+This is where we are currently at with GOV.UK. The vast majority of users on strong, stable connection will have seen improved performance from enabling HTTP/2. Unfortunately, some users may have seen performance drop if they use a connection with high packet loss. This is due to the HOL blocking that now exists at the network layer. This is a primary issue HTTP/3 sets out to solve.
+
+### HTTP/3
+To solve the HOL blocking issue at the network layer, HTTP/3 uses a completely new transport layer protocol called QUIC. QUIC initially stood for "Quick UDP Internet Connections", but the IETF now don't use it as an acronym. It is simply the name of the protocol. QUIC is built upon User Datagram Protocol (UDP). Since QUIC uses UDP, it isn't bound by the restrictions that TCP imposes. So it has now been designed so that the network layer now understands the the concept of independent streams, which was restricted to the application layer under HTTP/2:
```

Comment:
```suggestion
To solve the HOL blocking issue at the network layer, HTTP/3 uses a completely new transport layer protocol called QUIC. QUIC initially stood for "Quick UDP Internet Connections", but the IETF now don't use it as an acronym. It is simply the name of the protocol. QUIC is built upon the User Datagram Protocol (UDP), so isn't bound by the restrictions of TCP. It has been designed so that the network layer now understands the the concept of independent streams, which was restricted to the application layer under HTTP/2:
```